libs/login/WordPressLoginFlow/src/main/java/org/wordpress/android/login/SignupEmailFragment.java

```diff
@@ -264,7 +260,7 @@ public void onAvailabilityChecked(OnAvailabilityChecked event) {
                             if (event.isAvailable) {
                                 mLoginListener.showSignupMagicLink(event.value);
                             } else {
-                                AnalyticsTracker.track(AnalyticsTracker.Stat.SIGNUP_EMAIL_TO_LOGIN);
+                                mAnalyticsListener.trackAnalyticsSignIn(mAccountStore, mSiteStore, true);
```

Comment:
Shouldn't this be `trackSignupEmailToLogin()`?

Reply:
Yep.  I updated that statement in d20d82a.